### PR TITLE
TAS-1612/Intitate-Dispute

### PR DESCRIPTION
**Pull request by satanmourner**
**This branch is mainly related to `initiate dispute` modal on the contract page**
BUT including two small changes:
- [x] handled the null icon for `SearchDropdown` component
- [x] onClick prop in `Dot` component 

TODO:
- [x] category -> BE
- [x] deleting uploaded file issue (mandatory field)

### CODE REVIEW

**Review by marjan-ara — APPROVED** [review]

File: src/Nowruz/modules/contract/components/initiateDisputeModal/successInitiateDispute/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      inlineTitle={!isMobile}
+      customStyle="max-w-[400px]"
+      mobileCentered
+      headerDivider={false}
+      contentClassName="px-6 pb-6 md:px-12"
+    >
+      <div className="text-sm flex flex-col gap-4 leading-5 md:px-7 text-Gray-light-mode-700">
+        <p>
+          <span className="font-semibold text-Brand-700">{respondentName} </span>
+          has been notified of your dispute and has until April 18, 2024 00:00 UTC to respond. You will receive a
```
Comment (marjan-ara) [inline src/Nowruz/modules/contract/components/initiateDisputeModal/successInitiateDispute/index.tsx:31]: I think Data should be variable here

**Review by marjan-ara — CHANGES_REQUESTED** [review]

File: src/core/api/disputes/disputes.types.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Media } from '../media/media.types';
+import { Identity } from '../site/site.types';
+
+export interface IssueDisputePayload {
```
Comment (marjan-ara) [inline src/core/api/disputes/disputes.types.ts:4]: Based on our naming convention, the name of this interface is better to be `DisputeReq` and the response interface is better to be named `DisputeRes`

**Review by marjan-ara — CHANGES_REQUESTED** [review]

File: src/core/api/disputes/disputes.types.ts (hunk 2 above)
Comment (satanmourner) [reply src/core/api/disputes/disputes.types.ts:4]: Dispute is such a general name
We have a type for:
- issuing
- responding
Do you have other suggestions?

**Review by marjan-ara — APPROVED** [review]

**Comment by satanmourner** [comment]
@marjan-ara could you please re-review this PR? :)))

File: src/constants/DISPUTE_CATEGORY.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const DISPUTE_CATEGORY = [
+  { value: 'SCOPE_AND_CONTRACT_DISPUTES_ISSUES', label: 'Scope and Contract Disputes Issues' },
+  { value: 'INCOMPLETE_OR_UNSATISFACTORY_WORK', label: 'Incomplete or Unsatisfactory Work' },
+  { value: 'COMMUNICATION_PROBLEMS', label: 'Communication Problems' },
```
Comment (marjan-ara) [inline src/constants/DISPUTE_CATEGORY.ts:4]: 1. Isn't it better to get categories from API? 
2. I think not all categories are allowed for all users. for example, talent shouldn't be able to select `Incomplete or Unsatisfactory Work` as a category while initiating a dispute.

Comment (satanmourner) [reply src/constants/DISPUTE_CATEGORY.ts:4]: 1. it's static and it is not huge data, no need for API (I always prefer to reduce API calls)
2. no one told me about it and Minh designed one category list for both sides

**Pull request merged**